builder/src/lib.rs

```diff
@@ -51,18 +51,21 @@ fn run_cmd(cmd: &mut Command) -> io::Result<()> {
     }
 }
 
-fn run_cmd_stdout(cmd: &mut Command, input: Option<&[u8]>) -> io::Result<String> {
+pub fn run_cmd_stdout(cmd: &mut Command, input: Option<&[u8]>) -> io::Result<String> {
     cmd.stdin(Stdio::piped());
     cmd.stdout(Stdio::piped());
 
     let mut child = cmd.spawn()?;
-    if let (Some(mut stdin), Some(input)) = (child.stdin.take(), input) {
-        std::io::Write::write_all(&mut stdin, input)?;
+    if let (Some(stdin), Some(input)) = (child.stdin.take(), input) {
+        let mut writer = BufWriter::new(stdin);
+        writer.write_all(input)?;
+        writer.flush()?;
     }
     let out = child.wait_with_output()?;
     if out.status.success() {
         Ok(String::from_utf8_lossy(&out.stdout).into())
     } else {
+        println!("status ={:?}", out.status.code());
```

Comment:
remove this?

Reply:
Reverted to original and just made the symbol public.